src/build/__main__.py

```diff
@@ -117,7 +114,12 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         action='store_true',
         help='build a wheel',
     )
-    parser.add_argument('--outdir', '-o', metavar='dist', type=str, default=out, help='output directory')
+    parser.add_argument(
+        '--outdir',
+        '-o',
+        type=str,
+        help='output directory (defaults to sourcedir/dist)',
```

Comment:
I think it's cleaner if you put ``defaults to {sourcedir}/dist`` to show that dist is static, source dir is not. Personally, I would rename sourcedir to src_dir, more pythonic, easier to my eye 🤷

Reply:
Could change it to `srcdir` for consistency with `outdir`?

Reply:
Sounds good 👍